README.md

```diff
@@ -38,54 +34,52 @@ gov.au UI-Kit
 - [tags](http://uikit.apps.staging.digital.gov.au/packages/tags/tests/site/)
 - [text-inputs](http://uikit.apps.staging.digital.gov.au/packages/text-inputs/tests/site/)
 
-
 **[⬆ back to top](#content)**
 
-
 ----------------------------------------------------------------------------------------------------------------------------------------------------------------
 
-
 ## Build
 
-To build the UI-Kit you have to install [lerna](https://github.com/lerna/lerna) globally and run.
+Clone the UI-Kit project:
 
 ```shell
+git clone https://github.com/govau/uikit.git
```

Comment:
Should we really explain how to clone a git repo? If that's not clear I think we have to explain a lot more than that? Just thinking out loud here?